src/services/validations.js

```diff
@@ -0,0 +1,12 @@
+const joi = require('joi');
+
+const addRecipeValidation = {
+  name: joi.string().min(2).max(30).required(),
+  recipe: joi.string().min(50).required(),
+  img_url: joi.string().required(),
+  type: joi.boolean().required(),
+};
+
+module.exports = {
+  addRecipeValidation,
```

Comment:
as this is in the validation file, in future you could just name this `addRecipe` and then when you use it in `app.js` it will say `validate(valdations.addRecipe)` (at the moment you kind of repeat yourself